cardano-testnet/src/Testnet/Components/Configuration.hs

```diff
@@ -104,18 +105,12 @@ createSPOGenesisAndFiles testnetOptions startTime (TmpAbsolutePath tempAbsPath')
   -- TODO: Remove this rewrite.
  -- 50 second epochs
  -- Epoch length should be "10 * k / f" where "k = securityParam, f = activeSlotsCoeff"
-  H.rewriteJsonFile createStakedInitialGenesisFile $ J.rewriteObject
-      ( HM.insert "securityParam"          (toJSON @Int 5)    -- TODO: USE config p arameter
-      . HM.adjust
-          (J.rewriteObject
-              $ HM.adjust
-                (J.rewriteObject (HM.insert "major" (toJSON @Int 8)))
-                "protocolVersion"
-          )   "protocolParams"
-      . HM.insert "rho"                    (toJSON @Double 0.1)
-      . HM.insert "tau"                    (toJSON @Double 0.1)
-      . HM.insert "updateQuorum"           (toJSON @Int 2)
-      )
+  H.rewriteJsonFile createStakedInitialGenesisFile $ \o -> o
+    & L.key "securityParam" . L._Integer .~ 5
+    & L.key "rho" . L._Double  .~ 0.1
+    & L.key "tau" . L._Double  .~ 0.1
```

Comment:
those are wrong, they can't be at the top level. I am going to steal your lenses idea and put it into my PR.

Reply:
Where should they go instead?

Reply:
This is not a regression in any case.

Reply:
Inside `protocolParams`. This change just went overlooked, and I've stumbled upon it.